src/parcel.vue

```diff
@@ -0,0 +1,24 @@
+<script>
+export default {
+  props: {
+    config: Object
+  },
+  methods: {
+    noConfig(createElement) {
+      return createElement(
+        "div",
+        { class: "no-config" },
+        "<Parcel /> was mounted without a configuration object"
+      );
+    }
+  },
+  render(createElement) {
+    if (!!this.config) {
+      const appendTo = this.config.appendTo || "div";
+      return createElement(appendTo, { class: "parcel-container" });
```

Comment:
I'm not sure if using Vue's vdom for the parcel container is the right way to go. In React, we had to manually create the dom element with DOM apis, not with React APIs, in order for react to not get confused about nested react applications.

Reply:
@joeldenning, Got it. Thanks for the insight. 

Is it possible to append elements to the DOM instead of Vue's VDOM? From what I've been researching the only way to do so is by using the template component, instead of the render functions, but wouldn't this block me from allowing the users to choose which kind of element they want to be created?